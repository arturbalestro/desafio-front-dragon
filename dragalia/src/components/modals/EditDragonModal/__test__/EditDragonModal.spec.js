// @flow
import EditDragonModal from '../EditDragonModal';
import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';

const middlewares = [];
const mockStore = configureStore(middlewares);
const initialState = {};
const store = mockStore(initialState);

test('<EditDragonModal /> renders correctly', () => {
  const tree = renderer
    .create(<Provider store={store}><EditDragonModal /></Provider>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});