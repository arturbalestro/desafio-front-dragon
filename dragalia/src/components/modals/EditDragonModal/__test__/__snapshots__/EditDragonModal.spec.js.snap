// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditDragonModal /> renders correctly 1`] = `<div />`;
